etc/wingedhelix.cfg

```diff
@@ -1,6 +1,6 @@
 TMP_LFS='tmp07'
 PRM_LFS='prm07'
-SCR_LFS='scr01'
+SCR_LFS='tmp07'
```

Comment:
this to prevent making multiple changes in the code, if we don't want this because it is confusing i can revert this and change it in the code

Reply:
True, this may be a bit confusing, but there is no SCR on this system, so I'm Ok with using SCR_LFS pointing to TMP_LFS. Optionally, you can use the suggestion below.